content/en/docs/demo/feature-flags.md

```diff
@@ -10,12 +10,14 @@ feature flag service that supports [OpenFeature](https://openfeature.dev). Flag
 values are stored in the `demo.flagd.json` file. To enable a flag, change the
 `defaultVariant` value in the config file for a given flag to "on".
 
-| Feature Flag            | Service(s)      | Description                                                                                              |
-| ----------------------- | --------------- | -------------------------------------------------------------------------------------------------------- |
-| `adServiceFailure`      | Ad Service      | Generate an error for `GetAds` 1/10th of the time                                                        |
-| `cartServiceFailure`    | Cart Service    | Generate an error for `EmptyCart` 1/10th of the time                                                     |
-| `productCatalogFailure` | Product Catalog | Generate an error for `GetProduct` requests with product id: `OLJCESPC7Z`                                |
-| `recommendationCache`   | Recommendation  | Create a memory leak due to an exponentially growing cache. 1.4x growth, 50% of requests trigger growth. |
+| Feature Flag                  | Service(s)      | Description                                                                                              |
+| ----------------------------- | --------------- | -------------------------------------------------------------------------------------------------------- |
+| `adServiceFailure`            | Ad Service      | Generate an error for `GetAds` 1/10th of the time                                                        |
+| `cartServiceFailure`          | Cart Service    | Generate an error for `EmptyCart` 1/10th of the time                                                     |
+| `productCatalogFailure`       | Product Catalog | Generate an error for `GetProduct` requests with product id: `OLJCESPC7Z`                                |
+| `recommendationCache`         | Recommendation  | Create a memory leak due to an exponentially growing cache. 1.4x growth, 50% of requests trigger growth. |
+| `paymentServiceFailure`       | Paymentservice  | Generate an error when calling the charge method of the paymentservice.                                  |
```

Comment:
```suggestion
| `paymentServiceFailure`       | Payment Service  | Generate an error when calling the `charge` method                                  |
```

To make this similar to the other cells, I suggest removing the paymentservice (since it's in the previous column` and code-format the `charge` (if this is the name of the method)